Dockerfile

```diff
@@ -1,24 +1,28 @@
-FROM golang AS builder
-
-COPY . /go/src/github.com/rclone/rclone/
-WORKDIR /go/src/github.com/rclone/rclone/
-
-RUN \
-  CGO_ENABLED=0 \
-  make
-RUN ./rclone version
-
-# Begin final image
-FROM alpine:latest
-
+ARG ALPINE_VERSION=latest
+ARG GO_VERSION=latest
+ARG XCPUTRANSLATE_VERSION=v0.6.0
+ARG BUILDPLATFORM=linux/amd64
+
+FROM --platform=${BUILDPLATFORM} qmcgaw/xcputranslate:${XCPUTRANSLATE_VERSION} AS xcputranslate
```

Comment:
FYI this is a small Go program to convert for example `linux/arm/v7` to `GOARCH=arm` + `GOARM=7`